data-manager/src/test/groovy/org/orbisgis/datamanager/GroovyTest.groovy

```diff
@@ -90,4 +90,20 @@ class GroovyTest {
         assertTrue values.contains("LOADH2GIS.PUBLIC.TABLE1")
         assertTrue values.contains("LOADH2GIS.PUBLIC.TABLE2")
     }
-    }
\ No newline at end of file
+
+    @Test
+    void querySpatialTableMetaData() {
+        def h2GIS = H2GIS.open([databaseName: './target/loadH2GIS'])
+        h2GIS.execute("""
+                DROP TABLE IF EXISTS h2gis;
+                CREATE TABLE h2gis (id int, the_geom point);
+                INSERT INTO h2gis VALUES (1, 'POINT(10 10)'::GEOMETRY), (2, 'POINT(1 1)'::GEOMETRY);
+        """)
+        def concat = ""
+        h2GIS.rows "SELECT * FROM h2gis", {  meta ->
```

Comment:
This approach must changed in the future to support a syntax closed the spatialtable model as
h2GIS.getSpatialTable "h2gis" rows{meta ->}
and meta must offer spatial methods as getSRID